lib/routes/account.js

```diff
@@ -579,7 +599,8 @@ module.exports = function (
                           // value; Its stable, random, and will change on
                           // account reset.
                           wrapKb: account.wrapWrapKb,
-                          emailVerified: true
+                          emailVerified: true,
+                          tokenVerificationId: tokenVerificationId
```

Comment:
I seem to remember that we discussed this and decided it wasn't a big deal. Or did I dream that bit?